homeassistant/components/lutron_caseta/button.py

```diff
@@ -0,0 +1,72 @@
+"""Support for pico and keypad buttons."""
+from __future__ import annotations
+
+from homeassistant.components.button import ButtonEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import LutronCasetaDevice
+from .const import DOMAIN as CASETA_DOMAIN
+from .device_trigger import (
+    LEAP_TO_DEVICE_TYPE_SUBTYPE_MAP,
+    _lutron_model_to_device_type,
+)
+from .models import LutronCasetaData
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Lutron pico and keypad buttons."""
+    data: LutronCasetaData = hass.data[CASETA_DOMAIN][config_entry.entry_id]
+    bridge = data.bridge
+    bridge_devices = bridge.get_devices()
+    button_devices = bridge.get_buttons()
+
+    for button_device_id in button_devices:
+        # find and complete missing button names
+        button_device = button_devices[button_device_id]
+        ha_device = button_device
+        if (
+            "parent_device" in button_device
+            and button_device["parent_device"] is not None
+        ):
+            ha_device = bridge_devices[button_device["parent_device"]]
```

Comment:
```suggestion
        if (parent := button_device.get('parent_device')) is not None:
            ha_device = bridge_devices[parent]
```

It would be a bit cleaner to write it as this

or if the value for `parent_device` can never be `""` or false you can do

```python
        if (parent := button_device.get('parent_device')):
            ha_device = bridge_devices[parent]
```